### PR TITLE
[inductor][cpp] stabilize do_bench_cpu

### PR DESCRIPTION
**Pull request by jgong5**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #132208
* __->__ #131873

This PR stabilizes the `do_bench_cpu` by using milliseconds for warmup and benchmark runs, aligning with that of Trtion's do_bench.

cc @voznesenskym @penguinwu @EikanWang @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @ColinPeppler @amjames @desertfire @chauhang

### CODE REVIEW

**Review by leslie-fang-intel — APPROVED** [review]
LGTM

**Review by chunyuan-w — APPROVED** [review]

File: torch/_inductor/runtime/runtime_utils.py [hunk 1]
```diff
@@ -118,10 +119,29 @@ def load_triton():
     return triton_do_bench(*args, **kwargs)[0]
 
 
-def do_bench_cpu(fn, warmup=5, times=20):
+def do_bench_cpu(fn, warmup=5, times=20, min_duration=0.1):
```
Comment (eellison) [inline torch/_inductor/runtime/runtime_utils.py:122]: the triton / cuda do_bench takes in a duration to do benchmarking for and from that calculates # of warmup and iterations. is there a reason to invert it here ?

**Review by eellison — APPROVED** [review]

File: torch/_inductor/runtime/runtime_utils.py (hunk 1 above)
Comment (jgong5) [reply torch/_inductor/runtime/runtime_utils.py]: Thanks for the suggestion. Revised according to triton's definition.

**Review by leslie-fang-intel — APPROVED** [review]

**Pull request closed without merging**